packages/nova-lib/lib/redux.js

```diff
@@ -1,2 +1,3 @@
 export const Actions = {};
 export const Reducers = {};
+export const Middleware = {};
```

Comment:
In order to be spread when calling `applyMiddleware`, I believe this variable should be an array instead of an object? Did it work for you like that?

And just a detail, should we name this var at the singular or the plural (so we don't change it later)? Imo, it might makes more sense to have it named `Middlewares` (plu). I'm not English native, you surely know what makes better sense!